src/software/proto/message_translation/ssl_referee_test.cpp

```diff
@@ -73,6 +73,24 @@ TEST_P(RefboxGameStateTest, test_refbox_game_state)
 INSTANTIATE_TEST_CASE_P(
     RefboxGameStateTests, RefboxGameStateTest,
     ::testing::Values(
+        // common to yellow and blue
+        std::make_tuple(RefboxGameState::HALT, SSL_Referee_Command_HALT,
+                        TeamColour::YELLOW),
+        std::make_tuple(RefboxGameState::STOP, SSL_Referee_Command_STOP,
+                        TeamColour::YELLOW),
+        std::make_tuple(RefboxGameState::NORMAL_START, SSL_Referee_Command_NORMAL_START,
+                        TeamColour::YELLOW),
+        std::make_tuple(RefboxGameState::FORCE_START, SSL_Referee_Command_FORCE_START,
+                        TeamColour::YELLOW),
+        std::make_tuple(RefboxGameState::HALT, SSL_Referee_Command_HALT,
+                        TeamColour::BLUE),
+        std::make_tuple(RefboxGameState::STOP, SSL_Referee_Command_STOP,
+                        TeamColour::BLUE),
+        std::make_tuple(RefboxGameState::NORMAL_START, SSL_Referee_Command_NORMAL_START,
+                        TeamColour::BLUE),
+        std::make_tuple(RefboxGameState::FORCE_START, SSL_Referee_Command_FORCE_START,
+                        TeamColour::BLUE),
```

Comment:
not blocking, but might be a good use-case for gtest `Combine`

Reply:
I agree, but I don't think I can make the interface consistent with the rest of tests, so I don't think it's worth adding complexity for saving 16 lines

Reply:
I'm unsure what you mean by:

> I don't think I can make the interface consistent with the rest of tests,

But in any case, all good, approved.